src/main/java/seedu/budgetbaby/logic/commands/SetBudgetCommand.java

```diff
@@ -12,13 +12,14 @@
 public class SetBudgetCommand extends BudgetBabyCommand {
     public static final String COMMAND_WORD = "set-bg";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets the budget for the following months."
-            + "Parameters: BG_AMOUNT";
+        + "Parameters: BG_AMOUNT";
```

Comment:
Indentation for wrapped lines should be 8 spaces according to CS2103T's coding standard. Not a major issue.